auryadov/05.HTML/additional.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE HTML>
+<HTML>
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Форма регистрации</TITLE>
+	</HEAD>
+	<BODY>
+		<H1 align="center">Заполните анкету</H1>
+		<FORM ID="my_form1" name="pers_info" method="post" action="form.php">
+			<FIELDSET>
+				<LEGEND>Личные данные</LEGEND>
+				<INPUT type="text" name="surname" placeholder="Фамилия" pattern="^[А-Я] [а-я] {25}" required autofocus><BR>
```

Comment:
Need to remove white space form all pattern
ans 25- it equal 25 charset, but need not more that 25
Could you please align all pattern in this file
They do not work